planet-clicker/Assets/LibplanetUnity/Editor/LibplanetEditor.cs

```diff
@@ -20,6 +24,26 @@ public static void DeleteAllPlayer()
             DeleteAll(path);
         }
 
+        [MenuItem("Tools/Libplanet/Create Genesis")]
+        public static void CreateGenesisBlock()
+        {
+            if (
+                File.Exists(Agent.GenesisBlockPath) && 
+                !EditorUtility.DisplayDialog(
+                    "Create Genesis", 
+                    "Previous Genesis block has been found. Do you want to overwrite it?", 
```

Comment:
It's just question. Must the first character of *genesis* be uppercase?

Reply:
I don't have any idea about it. 

@kijun Could you suggest?

Reply:
It doesn't have to be upper case :)